Stylesheets/Components/ratings.css

```diff
@@ -0,0 +1,40 @@
+.rating-container {
+    width: fit-content;
+    height: fit-content;
+    gap: 1rem;
+    background-color: var(--color-background-dull);
+    padding: 8px;
+    border-radius: var(--container-border);
+}
+
+.rating-star{
+    margin:0;
+    position:relative;
+}
+
+.rating-container.badge {
+    position: static;
```

Comment:
This is not required since default position is static

Reply:
actually I am using the badge component class and this class is having position as absolute in badge.css so to change only this one property I am overriding the position property

Reply:
Good to know.
Code looks very clean & great view for grids!